src/index.ts

```diff
@@ -1,11 +1,18 @@
-import { configEmailBox } from "./integration-test/emailbox-integration";
+import { defineCustomElements } from '@ionic/pwa-elements/loader';
+import { configEmailBox, configEmailBoxEvents } from "./integration-test/emailbox-integration";
 import { configMultiDropdown } from "./integration-test/multidropdown-integration";
+import { configOcr } from './integration-test/ocr-integration';
 import { configObjectCounter } from "./integration-test/object-counter-integration";
 
-configEmailBox();
+defineCustomElements(window);
+
+//components configuration for testing purposes
 configMultiDropdown();
+configEmailBox();
+configEmailBoxEvents();
+configOcr();
 configObjectCounter();
 
 document.addEventListener('fieldChange', (event: any) => {
-  console.log('fieldChange.event', event.detail.value);
-});
\ No newline at end of file
+  console.log('fieldChange.event', event.detail);
```

Comment:
Shall we remove that Console.Log()?

Reply:
Well, to me it is useful to have it. This is for testing purposes to check which is the outcome from a field component